src/coprocessor/tracker.rs

```diff
@@ -150,11 +150,22 @@ impl Tracker {
     /// TiDB asks for ExecDetail to be printed in its log.
     pub fn get_item_exec_details(&self) -> kvrpcpb::ExecDetails {
         assert_eq!(self.current_stage, TrackerState::ItemFinished);
-        let is_slow_query = time::duration_to_sec(self.item_process_time) > SLOW_QUERY_LOWER_BOUND;
+        self.exec_details(self.item_process_time)
+    }
+
+    /// Get ExecDetail according to previous collected metrics.
+    /// TiDB asks for ExecDetail to be printed in its log.
+    pub fn get_exec_details(&self) -> kvrpcpb::ExecDetails {
+        assert_eq!(self.current_stage, TrackerState::AllItemFinished);
+        self.exec_details(self.total_process_time)
+    }
+
+    fn exec_details(&self, measure: Duration) -> kvrpcpb::ExecDetails {
+        let is_slow_query = time::duration_to_sec(measure) > SLOW_QUERY_LOWER_BOUND;
```

Comment:
I think we can remove this check to make it simple (TiDB almost always requests for a process time and keeping it will not improve performance as well)